File: Tareas/Unidad2/a2253332351_practica04/40_MayorDeTres.java
import java.util.Scanner;

public class MayorDeTres {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double n1, n2, n3, mayor;

        System.out.print("Introduce el primer numero: ");
        n1 = entrada.nextDouble();
        System.out.print("Introduce el segundo numero: ");
        n2 = entrada.nextDouble();
        System.out.print("Introduce el tercer numero: ");
        n3 = entrada.nextDouble();

        if (n1 >= n2 && n1 >= n3) {
            mayor = n1;
        } else if (n2 >= n1 && n2 >= n3) {
            mayor = n2;
        } else {
            mayor = n3;
        }

        System.out.println("El numero mayor es: " + mayor);
    }
}

File: Tareas/Unidad2/a2253332351_practica04/40_MayorDeTres.py
# MayorDeTres.py

n1 = float(input("Introduce el primer numero: "))
n2 = float(input("Introduce el segundo numero: "))
n3 = float(input("Introduce el tercer numero: "))

if n1 >= n2 and n1 >= n3:
    mayor = n1
elif n2 >= n1 and n2 >= n3:
    mayor = n2
else:
    mayor = n3

print(f"El numero mayor es: {mayor}")
